Composer/packages/client/src/components/BotRuntimeController/ErrorCallout.tsx

```diff
@@ -144,44 +137,25 @@ export const ErrorCallout: React.FC<IErrorCalloutProps> = (props) => {
   };
 
   return (
-    <Callout
-      setInitialFocus
-      ariaLabel={formatMessage(`{title}. {msg}`, { title: error.title, msg: error.message })}
-      data-testid={'errorCallout'}
-      gapSpace={0}
-      hidden={!visible}
-      target={target}
-      onDismiss={onDismiss}
-    >
-      <div css={calloutContainer}>
-        <p css={calloutLabel} id="callout-label-id">
-          {error.title}
-        </p>
-        <p css={calloutDescription} id="callout-description-id">
-          {buildErrorMessage(error)}
-          {error.linkAfterMessage != null && (
-            <Link href={error.linkAfterMessage.url} target={'_blank'}>
-              {error.linkAfterMessage.text}
-            </Link>
-          )}
-        </p>
-        {error.link != null && (
-          <p css={calloutLink}>
-            <Link href={error.link.url} target={'_blank'}>
-              {error.link.text}
-            </Link>
-          </p>
+    <div css={calloutContainer} data-testid="errorCallout">
+      <p css={calloutLabel} id="callout-label-id">
+        {error.title}
+      </p>
+      <p css={calloutDescription} id="callout-description-id">
+        {buildErrorMessage(error)}
+        {error.linkAfterMessage != null && (
+          <Link href={error.linkAfterMessage.url} target={'_blank'}>
+            {error.linkAfterMessage.text}
+          </Link>
         )}
-        <Stack
-          horizontal
-          tokens={{
-            childrenGap: 'm',
-          }}
-        >
-          <PrimaryButton text={formatMessage('Try again')} onClick={onTry} />
-          <DefaultButton text={formatMessage('Cancel')} onClick={onDismiss} />
-        </Stack>
-      </div>
-    </Callout>
+      </p>
+      {error.link != null && (
```

Comment:
if you are using != instead of strict !== you may as well just use ```!error.link && ...```

Reply:
Most of this is code was existing. Just had to break the bigger component into a smaller one